configuration/config.go

```diff
@@ -39,6 +39,8 @@ type IdentityBackendSettings struct {
 	SSLInsecureSkipVerify bool
 	MasterName            string
 	SentinelPassword      string
+	// TLS config of the backend
+	TLS TLS `json:"tls"`
```

Comment:
I wonder if wouldn't it be better to create the TLS configs on the root of the IdentityBackendSettings instead of in another TLS struct, since we already have the SSLInsecureSkipVerify config out there. 
What's the thinking process behind this decision?

Reply:
yes, indeed it was one of my concerns. I was ok with either one or the other, just made the changes